.github/workflows/pkg-config.yaml

```diff
@@ -54,7 +54,11 @@ jobs:
           unzip cargo-c-macos.zip -d ~/.cargo/bin
 
       - name: Install the library
-        run: make --file=Makefile.pkg-config PREFIX=${PREFIX} install
+        # NOTE: We set --libdir explicitly to avoid per-system/distro path components
+        #       that will complicate setting PKG_CONFIG_PATH/LD_LIBRARY_PATH.
+        run: >
```

Comment:
Is `>` vs `|` load bearing?

Reply:
Yeah, gotta "love" YAML....

`>` is "folded" style and will collapse `\n` into ` `, whereas `|` will keep the `\n`'s. We want the former in this case so the `CARGOFLAGS` env var and the command are the same line, as opposed to the usage on L52 where we want two separate lines (one for `curl` and one for `unzip`).

I always end up ref'ing https://yaml-multiline.info/ for this.